features/requestclose.yml.dist

```diff
@@ -0,0 +1,11 @@
+# Generated from: requestclose.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+status:
+  baseline: false
+  support:
+    chrome: "134"
+    chrome_android: "134"
+    edge: "134"
```

Comment:
Safari 18.4 also supports this (not sure where this data is generated from)

Reply:
Support data comes from @mdn/browser-compat-data. If Safari doesn't appear here, that's because 18.4 has not been released yet. The stable version is still 18.3.